src/style.css

```
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+.block {
+  height: 100px;
+}
+
+.red {
```

```suggestion
.block--red {
```
а ще краще 

```suggestion
.block--danger {
```
щоб не привязуватись до кольору